eo-runtime/src/main/java/org/eolang/PhDefault.java

```diff
@@ -354,8 +325,102 @@ private static void debug(final String msg) {
     /**
      * Padding according to current {@link #NESTING} level.
      * @return Padding string.
+     * @since 0.1
      */
     private static String padding() {
-        return String.join("", Collections.nCopies(NESTING.get(), "·"));
+        return String.join("", Collections.nCopies(PhDefault.NESTING.get(), "·"));
+    }
+
+    /**
+     * Make named attribute.
+     *
+     * @since 0.1
+     */
+    private static class Named {
```

Comment:
@l3r8yJ Can this be specific implementation of an `Attr` called, say, AtNamedDefault?
In constructor we create what we need:
```
AtNamedDefault(final Attr origin, final String name, final Phi phi) {
    this.named =  new AtNamed(
         String.format(
             "%s#%s",
             phi.getClass().getCanonicalName(),
             name
         ),
         String.format(
             "%s.%s",
             new Oname(phi),
             name
         ),
         phi,
         origin
    ); 
}
```
And overridden methods we will delegate calls to this.named.